.travis/run.sh

```diff
@@ -13,4 +13,6 @@ if [[ "${MACAPP_ENV}" == "system" ]]; then
 fi;
 
 source ~/.venv/bin/activate;
-tox --develop -- $TOX_FLAGS;
+
+ls .wheels;
```

Comment:
Yeah this was just debugging. But it might be useful to have it in future build logs, so I left it in.